ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -355,7 +356,7 @@ func (r *RayClusterReconciler) rayClusterReconcile(ctx context.Context, request
 	} else {
 		err = updateErr
 	}
-	if err != nil {
+	if err != nil || inconsistent {
```

Comment:
can you add a comment to explain why do we need to requeue the CR if the CR status is updated?